CoreVisualSchema/release/modules/ext/docs/CoreVisualSchema/src/au/gov/asd/tac/constellation/graph/schema/visual/complete-with-schema.md

```diff
@@ -0,0 +1,92 @@
+# Complete With Schema
+
+<table class="table table-striped">
+<colgroup>
+<col style="width: 25%" />
+<col style="width: 25%" />
+<col style="width: 25%" />
+<col style="width: 25%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th>Constellation Action</th>
+<th>Keyboard Shortcut</th>
+<th>User Action</th>
+<th>Menu Icon</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>Run Complete With Schema</td>
+<td>F5</td>
+<td>Tools -&gt; Complete With Schema</td>
+<td>
+<img src="../ext/docs/CoreVisualSchema/src/au/gov/asd/tac/constellation/graph/schema/visual/resources/completeWithSchema.png" width="16" height="16" />
+</td>
+</tr>
+</tbody>
+</table>
+
+Complete with Schema is a plugin that can be run to update graph attributes to the
+match the expected values for the graph's schema. Each graph has a schema which 
+defines which attributes are included in that graph. This plugin can be run via 
+the Tools menu and is also run by default after using some Data Access Plugins or
+running some Analytics. 
+
+## Schemas
```

Comment:
I note that you make reference to the Schemas page later on (though I would use a link) but given that other page exists, I wonder if we're best off just linking to that page there for more information on schemas and just then omitting this section which is otherwise just conveying information found on that page

Reply:
The schema page didn't exist yet when I wrote it but I can remove and just link the new page instead